public/app/themes/justice/src/css/global.css

```diff
@@ -3129,14 +3129,18 @@ div#content-wrapper div.container-wrapper div#content article div.article form#g
     color: #4E6280;
 }
 
+/* Migration edit - see justice/src/components/simple-definition-list-blocks */
+
 /* Civil Procedure Rules definition list (dl)*/
-#content div.article dl.glossary dt.expression {
+/* #content div.article dl.glossary dt.expression {
     font-weight: bold;
 }
 
 #content div.article dl.glossary dd.meaning {
     margin-left: 20px;
-}
+} */
```

Comment:
If this is commented out can we just remove it entirely?

Reply:
Good point - I've commented out rather than delete in this file because it's legacy code and potentially I'll need to read it quickly in the future. I'll delete the commented blocks after UAT & migration.